saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsMeta.jsp

```diff
@@ -0,0 +1,211 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+
+
+<spring:message code="judgmentDetails.meta.keywords" var="detailsMetaKeywords" />
+
+<%-- JudgmentType --%>
+<c:if test="${!empty judgment.judgmentType}" >
+    <c:if test="${!empty detailsMetaKeywords}" >
+        <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords += ', '}" />
```

Comment:
Takie wyrażenia powodują u mnie:
Caused by: org.apache.jasper.JasperException: /WEB-INF/view/details/judgmentDetailsMeta.jsp (line: 10, column: 8) "${detailsMetaKeywords += ', '}" contains invalid expression(s): javax.el.ELException: Failed to parse the expression [${detailsMetaKeywords += ', '}]